tool/tctl/common/plugin/awsic.go

```diff
@@ -0,0 +1,208 @@
+	"github.com/gravitational/teleport/api/types"
+	apicommon "github.com/gravitational/teleport/api/types/common"
+	"github.com/gravitational/teleport/lib/utils"
+	toolcommon "github.com/gravitational/teleport/tool/common"
+)
+
+type awsICArgs struct {
+	cmd                  *kingpin.CmdClause
+	defaultOwners        []string
+	scimToken            string
+	scimURL              string
+	region               string
+	arn                  string
+	useSystemCredentials bool
+	userOrigin           string
+	userLabels           toolcommon.Labels
+	groupNameFilters     []string
+	accountNameFilters   []string
+	accountIDFilters     []string
+}
+
+func (a *awsICArgs) validate() error {
+	if !a.useSystemCredentials {
+		return trace.BadParameter("only AWS Local system credentials are supported")
+	}
+	return nil
+}
+
+// parseAWSICNameFilters validates that all elements of the supplied [names] slice
+// are valid regexes or globs and wraps them in [types.AWSICResourceFilter]s for
+// inclusion in a [types.PluginAWSICSettings].
+//
+// We are using a manual validator here rather than the canonical one defined
+// in the AWS IC integration itself, because those filter tools are not
+// available to OSS builds of tctl.
+func parseAWSICNameFilters(names []string) ([]*types.AWSICResourceFilter, error) {
+	var filters []*types.AWSICResourceFilter
+	for _, n := range names {
+		if _, err := utils.CompileExpression(n); err != nil {
+			return nil, trace.Wrap(err)
+		}
+		filters = append(filters, &types.AWSICResourceFilter{
+			Include: &types.AWSICResourceFilter_NameRegex{NameRegex: n},
+		})
+	}
+	return filters, nil
+}
+
+func (a *awsICArgs) parseGroupFilters() ([]*types.AWSICResourceFilter, error) {
+	return parseAWSICNameFilters(a.groupNameFilters)
+}
+
+func (a *awsICArgs) parseAccountFilters() ([]*types.AWSICResourceFilter, error) {
+	filters, err := parseAWSICNameFilters(a.accountNameFilters)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	for _, id := range a.accountIDFilters {
+		filters = append(filters, &types.AWSICResourceFilter{
+			Include: &types.AWSICResourceFilter_Id{Id: id},
+		})
+	}
+
+	return filters, nil
+}
+
+func (a *awsICArgs) parseUserFilters() ([]*types.AWSICUserSyncFilter, error) {
+	var result []*types.AWSICUserSyncFilter
+
+	labels := make(toolcommon.Labels)
+	maps.Copy(labels, a.userLabels)
+	if a.userOrigin != "" {
+		labels[types.OriginLabel] = a.userOrigin
+	}
+
+	if len(labels) > 0 {
+		result = append(result, &types.AWSICUserSyncFilter{Labels: labels})
+	}
+
+	return result, nil
+}
+
+func (p *PluginsCommand) initInstallAWSIC(parent *kingpin.CmdClause) {
+	p.install.awsIC.cmd = parent.Command("awsic", "Install an AWS Identity Center integration.")
+	cmd := p.install.awsIC.cmd
+	cmd.Flag("default-owner", "List of Teleport users that are default owners for the imported access lists. Multiple flags allowed.").Required().StringsVar(&p.install.awsIC.defaultOwners)
```

Comment:
will it be better ux by supporting comma separate values over multiple flag? not sure what we do for other existing commands.